### PR TITLE
Populate Firestore with specific roster's courses

### PR DESCRIPTION
**Pull request by rohanmaheshwari430**
### Summary 

In this PR, we write a script that will populate the `courses` collection with the courses of the latest roster. It will also update `available-rosters-for-course` collection by adding the latest roster to the rosters array. Finally, it will update the `crseid-to-catalognbr` collection for any courses whose crseId may have changed in the latest roster (unlikely but still a good measure to have). 

### Test Plan 

Performed checks using Firestore's query builder to ensure that each modified collection contained the appropriate data and that their schema was in line with existing data. Checked various courses and ensured that their offerings were in line with what is in the firestore.

### CODE REVIEW

**Review by zachary0kent — CHANGES_REQUESTED** [review]
Is there any way we can refactor this to that the original population script calls the functions we added here?

File: scripts/population/populate-specific-roster.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+      `There are ${subjectCourses.length} courses in ${subject.subject} in ${subject.roster}.`
+    );
+    console.log(`We fetched ${subjectCount} out of ${rosterSubjectPairs.length} subjects`);
+  }
+
+  console.log(`There are ${coursesCount} total courses.`);
+};
+
+/* Enter the desired roster as a string argument (e.g 'FA21'). Fetches latest roster data by default */
+populate();
```
Comment (zachary0kent) [inline scripts/population/populate-specific-roster.ts:108]:
```suggestion
import parseArgs from minimist 

/* Enter the desired roster as a string argument (e.g 'FA21'). Fetches latest roster data by default */
populate(parseArgs(process.argv.slice(2)).roster);
```

Comment (zachary0kent) [reply scripts/population/populate-specific-roster.ts:108]: This way we can pass the roster in as a command line argument (we should also move the import to the top of the file)

**Review by zachary0kent — CHANGES_REQUESTED** [review]
Just one suggestion!

File: scripts/population/populate-specific-roster.ts (hunk 1 above)
Comment (rohanmaheshwari430) [reply scripts/population/populate-specific-roster.ts:108]: BETSKI

**Review by zachary0kent — APPROVED** [review]
Looks good!

**Pull request merged**